Numdiff build error on MAC

```
In file included from /Users/boyceg/sfw/autoibamr/install-release/tmp/unpack/numdiff-5.9.0/error.c:33:
/Users/boyceg/sfw/autoibamr/install-release/tmp/unpack/numdiff-5.9.0/gettext.h:25:11: fatal error: 'libintl.h' file not found
# include <libintl.h>
          ^~~~~~~~~~~
In file included from /Users/boyceg/sfw/autoibamr/install-release/tmp/unpack/numdiff-5.9.0/cmpfns.c:24:
In file included from /Users/boyceg/sfw/autoibamr/install-release/tmp/unpack/numdiff-5.9.0/numdiff.h:23:
/Users/boyceg/sfw/autoibamr/install-release/tmp/unpack/numdiff-5.9.0/system.h:270:10: fatal error: 'libintl.h' file not found
#include <libintl.h>
         ^~~~~~~~~~~
In file included from /Users/boyceg/sfw/autoibamr/install-release/tmp/unpack/numdiff-5.9.0/arith.c:21:
In file included from /Users/boyceg/sfw/autoibamr/install-release/tmp/unpack/numdiff-5.9.0/numdiff.h:23:
/Users/boyceg/sfw/autoibamr/install-release/tmp/unpack/numdiff-5.9.0/system.h:270:10: fatal error: 'libintl.h' file not found
#include <libintl.h>
         ^~~~~~~~~~~
1 error generated.
In file included from /Users/boyceg/sfw/autoibamr/install-release/tmp/unpack/numdiff-5.9.0/flags.c:20:
In file included from /Users/boyceg/sfw/autoibamr/install-release/tmp/unpack/numdiff-5.9.0/numdiff.h:23:
/Users/boyceg/sfw/autoibamr/install-release/tmp/unpack/numdiff-5.9.0/system.h:270:10: fatal error: 'libintl.h' file not found
#include <libintl.h>
         ^~~~~~~~~~~
1 error generated.
make: *** [error.o] Error 1
1 error generated.
make: *** [cmpfns.o] Error 1
make: *** [flags.o] Error 1
1 error generated.
make: *** [arith.o] Error 1
In file included from /Users/boyceg/sfw/autoibamr/install-release/tmp/unpack/numdiff-5.9.0/exitfail.c:20:
/Users/boyceg/sfw/autoibamr/install-release/tmp/unpack/numdiff-5.9.0/system.h:270:10: fatal error: 'libintl.h' file not found
#include <libintl.h>
         ^~~~~~~~~~~
1 error generated.
make: *** [exitfail.o] Error 1
In file included from /Users/boyceg/sfw/autoibamr/install-release/tmp/unpack/numdiff-5.9.0/getopt.c:53:
/Users/boyceg/sfw/autoibamr/install-release/tmp/unpack/numdiff-5.9.0/gettext.h:25:11: fatal error: 'libintl.h' file not found
# include <libintl.h>
          ^~~~~~~~~~~
1 error generated.
make: *** [getopt.o] Error 1
Failure with exit status: 2
Exit message: There was a problem building numdiff 5.9.0.
```

--- a/IBAMR-toolchain/packages/numdiff.package
+++ b/IBAMR-toolchain/packages/numdiff.package
@@ -5,7 +5,7 @@ PACKING=.tar.gz
 CHECKSUM=794461a7285d8b9b1f2c4a8149889ea6
 BUILDCHAIN=autotools
 
-CONFOPTS=""
+CONFOPTS="--enable-nls=no --enable-gmp=no --enable-optimization"
 
 INSTALL_PATH=${INSTALL_PATH}/${NAME}
 

